app/controllers/microposts_controller.rb

```diff
@@ -0,0 +1,38 @@
+class MicropostsController < ApplicationController
+  before_action :logged_in_user, only: %i(create destroy)
+  before_action :correct_user, only: :destroy
+
+  def create
+    @micropost = current_user.microposts.build micropost_params
+    if @micropost.save
+      flash[:success] = t ".flash_success"
+      redirect_to root_url
```

Comment:
```suggestion
      redirect_to root_path
```
nếu ở trong các controller thì nên dùng path. k nên dùng url
e biết vì sao k

Reply:
vì dùng path, routes điều hướng đến 1 controller action tương ứng rồi nên k cần dùng url đúng k ạ?